### PR TITLE
modified code to address issue111

### PR DESCRIPTION
**Pull request by naqvia**
#### What feature is being added or bug is being addressed?
https://github.com/diskin-lab-chop/AutoGVP/issues/111

#### What GitHub issue does your pull request address?
https://github.com/diskin-lab-chop/AutoGVP/issues/111

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
So, I was able to get the correct call and its because we had the final call being replaced of intervar being replaced by clinVar. I have changed a critical if statement that was doing that.

Please run to ensure the 0 stars are being replaced 

#### Documentation Checklist

- [ ] The function has examples to showcase the usage 
- [ ] Added a vignette

### CODE REVIEW

File: AutoGVP/01-annotate_variants_CAVATICA_input.R [hunk 1]
```diff
@@ -355,9 +355,8 @@ combined_tab_with_vcf_intervar <- autopvs1_results %>%
 
 ## merge tables together (clinvar and intervar) and write to file
 master_tab <- clinvar_anno_intervar_vcf_df %>%
-  full_join(combined_tab_with_vcf_intervar[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_with_vcf_intervar))], by = "vcf_id") %>%
-  # full_join(combined_tab_for_intervar_cc_removed[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_for_intervar_cc_removed))], by = "vcf_id") %>%
-  left_join(submission_final_df, by = "vcf_id")
+  full_join(combined_tab_with_vcf_intervar[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_with_vcf_intervar))], by = "vcf_id") # %>%
+# left_join(submission_final_df, by = "vcf_id")
```
Comment (jharenza) [inline AutoGVP/01-annotate_variants_CAVATICA_input.R:359]:
```suggestion
  full_join(combined_tab_with_vcf_intervar[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_with_vcf_intervar))], by = "vcf_id")
```

File: AutoGVP/01-annotate_variants_custom_input.R [hunk 2]
```diff
@@ -375,9 +364,9 @@ combined_tab_with_vcf_intervar <- autopvs1_results %>%
 
 ## merge tables together (clinvar and intervar) and write to file
 master_tab <- clinvar_anno_intervar_vcf_df %>%
-  left_join(combined_tab_with_vcf_intervar[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_with_vcf_intervar))], by = "vcf_id") %>%
-  #  left_join(combined_tab_for_intervar_cc_removed[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_for_intervar_cc_removed))], by = "vcf_id") %>%
-  left_join(submission_final_df, by = "vcf_id")
+  left_join(combined_tab_with_vcf_intervar[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_with_vcf_intervar))], by = "vcf_id") # %>%
+# left_join(submission_final_df, by = "vcf_id")
```
Comment (jharenza) [inline AutoGVP/01-annotate_variants_custom_input.R:368]:
```suggestion
  left_join(combined_tab_with_vcf_intervar[, grepl("vcf_id|intervar_adjusted|evidence|InterVar:|criterion|final_call", names(combined_tab_with_vcf_intervar))], by = "vcf_id")
```

File: AutoGVP/01-annotate_variants_custom_input.R [hunk 3]
```diff
@@ -410,12 +400,12 @@ master_tab <- full_join(master_tab, entries_for_cc_in_submission, by = "vcf_id")
   full_join(entries_for_cc_in_submission_w_intervar[c("vcf_id", "Intervar_evidence")], by = "vcf_id") %>%
   dplyr::mutate(
     Intervar_evidence = coalesce(Intervar_evidence.y, Intervar_evidence.x),
-    ClinVar_ClinicalSignificance = coalesce(ClinicalSignificance.x, ClinicalSignificance.y)
+    # ClinVar_ClinicalSignificance = coalesce(ClinicalSignificance.x, ClinicalSignificance.y)
```
Comment (jharenza) [inline AutoGVP/01-annotate_variants_custom_input.R:403]:
```suggestion
```

File: AutoGVP/01-annotate_variants_custom_input.R [hunk 4]
```diff
@@ -410,12 +400,12 @@ master_tab <- full_join(master_tab, entries_for_cc_in_submission, by = "vcf_id")
...
   ) %>%
   dplyr::select(
     -final_call.x, -final_call.y,
-    -Intervar_evidence.x, -Intervar_evidence.y,
-    -ClinicalSignificance.x, -ClinicalSignificance.y
+    -Intervar_evidence.x, -Intervar_evidence.y # ,
+    #-ClinicalSignificance.x, -ClinicalSignificance.y
```
Comment (jharenza) [inline AutoGVP/01-annotate_variants_custom_input.R:408]:
```suggestion
    -Intervar_evidence.x, -Intervar_evidence.y
```

**Review by jharenza — COMMENTED** [review]
Can you remove all of the commented sections I deleted

**Review by jharenza — COMMENTED** [review]
also need to change line 373/374 in cavatica workflow

**Comment by naqvia** [comment]
Made necc changes. Works as expected. cc @jharenza

File: AutoGVP/01-annotate_variants_custom_input.R [hunk 5]
```diff
@@ -410,12 +399,11 @@ master_tab <- full_join(master_tab, entries_for_cc_in_submission, by = "vcf_id")
...
   dplyr::mutate(
     Intervar_evidence = coalesce(Intervar_evidence.y, Intervar_evidence.x),
-    ClinVar_ClinicalSignificance = coalesce(ClinicalSignificance.x, ClinicalSignificance.y)
   ) %>%
   dplyr::select(
     -final_call.x, -final_call.y,
-    -Intervar_evidence.x, -Intervar_evidence.y,
-    -ClinicalSignificance.x, -ClinicalSignificance.y
+    -Intervar_evidence.x, -Intervar_evidence.y # ,
+    #-ClinicalSignificance.x, -ClinicalSignificance.y
```
Comment (jharenza) [inline AutoGVP/01-annotate_variants_custom_input.R:406]: I don't think this was removed

**Review by jharenza — APPROVED** [review]
👍

**Pull request merged**